tests/files.rs

```diff
@@ -125,22 +125,28 @@ fn generate_tests(glob: &str) -> Vec<Trial> {
             test_terms_encoding: false,
         };
         let should_fail = run.should_fail();
+        // Marking as subsumed and non extractable is not supported for eqsat values with the term encoding
```

Comment:
Delete is also unsupported in the term encoding with eqsat return values.

Reply:
I'll make this work when I re-write the terms encoding again (: